docs/start/qiskit-versioning.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Qiskit Versioning and Support Policy
+description: The versioning and support policy for Qiskit.
+
+---
+# Qiskit Versioning
+Qiskit version numbers follow [Semantic Versioning](https://semver.org/).
+The version number is comprised of 3 primary components, the major, minor, and
+patch versions. For a version number `X.Y.Z` where `X` is the major version,
+`Y` is the minor version, and `Z` is the patch version.
+
+Breaking API changes are reserved for major version releases. The **minimum**
+period between major version releases is one year. Minor versions will be
+periodically (currently every three months) published for the current major
+version which add new features and bug fixes. For the most recent minor version,
+there will also be new patch versions published as bugs are identified and fixed
+on that release series. A tentative release schedule is included below:
+
+![](/images/qiskit-versioning/release_schedule.png)
```

Comment:
The file is in the release place, but link is wrong. Btw you can preview things locally: https://github.com/Qiskit/documentation?tab=readme-ov-file#preview-the-docs-locally

```suggestion
![](/images/start/qiskit-versioning/release_schedule.png)
```